scripts/release_ppa.sh

```diff
@@ -51,17 +58,39 @@ is_release() {
     [[ "${branch}" =~ ^v[0-9]+(\.[0-9]+)*$ ]]
 }
 
-keyid=379F4801D622CDCF
-email=builds@ethereum.org
+# source keyid and email from .release_ppa_auth
+if [[ -e .release_ppa_auth ]]
+then
+    # shellcheck source=/dev/null
+    source .release_ppa_auth
+fi
+
+[[ "$LAUNCHPAD_KEYID" != "" && "$LAUNCHPAD_EMAIL" != "" ]] ||\
+    fail "Error: Couldn't find variables \$LAUNCHPAD_KEYID or \$LAUNCHPAD_EMAIL in sourced file .release_ppa_auth (check top comment in $0 for more information)."
+
 packagename=solc
 
-static_build_distribution=impish
+# This needs to be a still active release
+static_build_distribution=focal
+
+DISTRIBUTIONS="focal jammy kinetic"
 
-DISTRIBUTIONS="focal impish jammy kinetic"
+function checkDputEntries () {
```

Comment:
I'd stick to the `function f {}` syntax. Both for consistency (I was cleaning that up at some point: #12259) and because it's more universal (`function f() {}` does not work in `sh`).

```suggestion
function checkDputEntries {
```